index.html

```diff
@@ -8,47 +8,324 @@
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <title>Desktop version</title>
     <link rel="stylesheet" href="style.css" />
-    <title>Setup and mobile version</title>
   </head>
   <body>
-    <header>
-      <nav class="nav">
-        <h2 class="name">Mucyo</h2>
-
-        <i class="fa fa-bars"></i>
+    <div class="container">
+      <nav>
+        <div class="logo">LOGO</div>
```

Comment:
[ OPTIONAL ] Please align navigation items vertically, and use the font which is defined in the design. Kindly see the below screenshot for reference.
![heading](https://user-images.githubusercontent.com/35504149/130317832-bb47b318-d0ab-4468-90f3-f535bd039f13.png)